commands/dht.go

```diff
@@ -0,0 +1,163 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	cmds "gx/ipfs/QmQtQrtNioesAWtrx8csBvfY37gTe94d6wQ3VikZUjxD39/go-ipfs-cmds"
+	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	pstore "gx/ipfs/QmRhFARzTHcFh8wUxwN5KvyTGq73FLC65EfFAhz8Ng7aGb/go-libp2p-peerstore"
+	notif "gx/ipfs/QmWaDSNoSdSXU9b6udyaq9T8y6LkzMwqWxECznFqvtcTsk/go-libp2p-routing/notifications"
+	cmdkit "gx/ipfs/Qmde5VP1qUkyQXKCfmEUA7bP64V2HAptbJ7phuPp7jXWwg/go-ipfs-cmdkit"
+)
+
+const (
+	dhtVerboseOptionName   = "verbose"
```

Comment:
I kept the same UI from ipfs, including this verbose mode.  It looks like it will be helpful for debugging (an explicit goal of this dht command) but it also departs from behavior in other commands.  Input welcome.